app/src/views/search-engine/components/articles-list/styles.js

```diff
@@ -9,7 +9,8 @@ const styles = makeStyles(() => ({
   },
   cardContent: {
     whiteSpace: 'pre-wrap',
-    marginBottom: 15
+    marginBottom: 15,
+    maxHeight: '20vh'
```

Comment:
TIL about `vh` as a CSS unit. vcool!

Reply:
Hahahaha yeah!! There are a bunch of [units in CSS](https://www.w3schools.com/cssref/css_units.asp).